src/style.css

```diff
@@ -1 +1,72 @@
-/* Styles go here */
+/* stylelint-disable string-quotes */
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap');
+
+* {
+  padding: 0;
+  margin: 0;
+}
```

Comment:
If you want to reset default margins and padding, do it for each tag individually.
Don't use selector `*`